### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by nuria-opusvl**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can you please follow OCA's README template (must be in RST formato for Odoo to read it): https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

**Comment by nuria-opusvl** [comment]
Pedro, this one isn't following it either: https://github.com/OCA/l10n-united-kingdom

**Comment by pedrobaeza** [comment]
OCA's template is for modules, not for repository one. Odoo is also not reading the top level README.md, but each module README.

**Comment by nuria-opusvl** [comment]
Aah, ok, I'll try.

**Comment by nuria-opusvl** [comment]
I'm just learning about markdown, so I'm not really sure about what I've down with the link urls to be honest. I hope it's ok, but if it isn't you'll have to show me the right way, please.

**Comment by pedrobaeza** [comment]
Sorry, but that was because I'll tell you about the template and to use RST format (RestructuredText). It's the only one supported by Odoo. You should rename the file to .rst and follow the template.

**Comment by nuria-opusvl** [comment]
aah, so it's not md anymore, it's rst. Should I still call it README?

**Comment by pedrobaeza** [comment]
In repository, it's README.md, because it's the native format of GitHub and it's the only one that it's going to read this one. And on modules, README.rst with RST format.

**Comment by nuria-opusvl** [comment]
All tests passed, shall I merge the pull request?

File: l10n_uk_sale_report/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+   :alt: License: AGPL-3
+
+=====================
+# l10n_uk_sale_report
+=====================
+
+Odoo module to introduce UK format on reports, that can fit in standard window envelopes in the UK.
+
+Installation
+=====================
```
Comment (pedrobaeza) [inline l10n_uk_sale_report/README.rst]: Underline length should be the same as the text length

File: l10n_uk_sale_report/README.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+To install this module, you need to:
+* Clone it to your addons directory
+* Restart Odoo service
+
+Configuration
+=====================
```
Comment (pedrobaeza) [inline l10n_uk_sale_report/README.rst]: The same

File: l10n_uk_sale_report/README.rst [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+No configuration needed
+
+Usage
+=====================
```
Comment (pedrobaeza) [inline l10n_uk_sale_report/README.rst]: The same

File: l10n_uk_sale_report/README.rst [hunk 4]
```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+# l10n_uk_sale_report
```
Comment (pedrobaeza) [inline l10n_uk_sale_report/README.rst]: Remove `#`. It's not a valid marker

File: l10n_uk_sale_report/README.rst [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+Configuration
+=====================
+
+No configuration needed
+
+Usage
+=====================
+
+## Sales Orders
```
Comment (pedrobaeza) [inline l10n_uk_sale_report/README.rst]: `##` is not a valid RST marker

File: l10n_uk_sale_report/README.rst [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+## Invoices
+* From print drop down menu, option available to print the sales order in UK format
+* *Print* button defaults to UK format on printing
+
+## Customers
+* From print drop down menu, option available to print account statements in UK format.
+
+Known issues / Roadmap
+=====================
```
Comment (pedrobaeza) [inline l10n_uk_sale_report/README.rst]: The same about length

File: l10n_uk_sale_report/README.rst [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+TODO: *Send by Email* button needs to default to UK format.
+Manual Workaround -> Technical features flag on for your user. Refresh and go to *Settings > Technical > Emails > Templates*. Open up *Sales Order - Send by Email*, edit. Go to *Advanced Settings* tab and choose *Sales Order (UK format)* from the list of options on *Optional report to print and attach*.
+
+
+TODO: *Send by Email* button needs to default to UK format.
+Manual Workaround -> Technical features flag on for your user. Refresh and go to *Settings > Technical > Emails > Templates*. Open up *Invoice - Send by Email*, edit. Go to *Advanced Settings* tab and choose *Invoice (UK format)* from the list of options on *Optional report to print and attach*.
+
+Bug Tracker
+=====================
```
Comment (pedrobaeza) [inline l10n_uk_sale_report/README.rst]: The same about length

File: l10n_uk_sale_report/README.rst [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{l10n-united-kingdom}/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/{l10n-united-kingdom}/issues/new?body=module:%20{l10n_uk_sale_report}%Aversion:%20{1.0}%0A%0A**Steps%20to%reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=====================
```
Comment (pedrobaeza) [inline l10n_uk_sale_report/README.rst]: The same about length

**Comment by pedrobaeza** [comment]
Nuria, how many times we need to tell you that the OCA review process forces to have at least 2 approvals and 5 days or 3 approvals?

**Comment by nuria-opusvl** [comment]
Pedro,

I hope this is the very last time you are condescending to me.

**Comment by pedrobaeza** [comment]
Sorry if you are upset, but why then you ask again and again the same thing?

**Comment by nuria-opusvl** [comment]
Are you saying I shouldn't ask?

**Comment by pedrobaeza** [comment]
No, I'm saying that you shouldn't repeat the same question across PRs. The answer is the same. Here is some of the times we had already told you:
- https://github.com/OCA/l10n-united-kingdom/pull/4#issuecomment-139097951
- https://github.com/OCA/l10n-united-kingdom/pull/3#issuecomment-137778346
- https://github.com/OCA/l10n-united-kingdom/pull/4#issuecomment-138482421

**Comment by nuria-opusvl** [comment]
Perhaps you should work on the assumption that not every body is the same nor learns at the same path, and that respect is a better approach than condescension.

I hope I will not have to repeat myself on this statement.

**Comment by pedrobaeza** [comment]
OK, let's move with this PR. All is OK, so you have the first :+1:

**Comment by dreispt** [comment]
:+1:

**Comment by colinnewell** [comment]
:+1:

**Pull request merged**